content/en/docs/getting-started-guides/windows/_index.md

```diff
@@ -3,399 +3,1025 @@ title: Using Windows Server Containers in Kubernetes
+![alt_text](flannel-master-kubectl-get-ds.png "flannel master kubectl get ds screen capture")
+
+#### Join Windows Worker
+
+In this section we'll cover configuring a Windows node from scratch to join a cluster on-prem. If your cluster is on a cloud you'll likely want to follow the cloud specific guides in the next section.
+
+#### Preparing a Windows Node
+{{< note >}}
+All code snippets in Windows sections are to be run in a PowerShell environment with elevated permissions (Admin).
+{{< /note >}}
+
+1. Install Docker (requires a system reboot)
+
+Kubernetes uses [Docker](https://www.docker.com/) as its container engine, so we need to install it. You can follow the [official Docs instructions](https://docs.microsoft.com/en-us/virtualization/windowscontainers/manage-docker/configure-docker-daemon#install-docker), the [Docker instructions](https://store.docker.com/editions/enterprise/docker-ee-server-windows), or try these steps:
 
-For overlay, OVS on Windows requires a transparent docker network to function properly. Please use the following to create a transparent docker network which will be used by OVS. From powershell:
+```powershell
+PS C:\Users\Administrator\Install-Module -Name DockerMsftProvider -Repository PSGallery -Force
+PS C:\Users\Administrator\Install-Package -Name Docker -ProviderName DockerMsftProvider
+PS C:\Users\Administrator\Restart-Computer -Force
 ```
-docker network create -d transparent --gateway $GATEWAY_IP --subnet $SUBNET `
-    -o com.docker.network.windowsshim.interface="$INTERFACE_ALIAS" external
+
+If you are behind a proxy, the following PowerShell environment variables must be defined:
+
+```powershell
+[Environment]::SetEnvironmentVariable("HTTP_PROXY", "http://proxy.example.com:80/", [EnvironmentVariableTarget]::Machine)
+[Environment]::SetEnvironmentVariable("HTTPS_PROXY", "http://proxy.example.com:443/", [EnvironmentVariableTarget]::Machine)
 ```
-Where $SUBNET is the minion subnet which will be used to spawn pods on (the one which will be used by kubernetes), $GATEWAY_IP is the first IP of the $SUBNET and $INTERFACE_ALIAS is the interface used for creating the overlay tunnels (must have connectivity with the rests of the OVN hosts).
-Example:
+
+If after reboot you may see the following error:
+
+![alt_text](windows-docker-error.png "windows docker error screen capture")
+If so then you need to restart the docker service manually:
+
+```powershell
+Start-Service docker
 ```
-docker network create -d transparent --gateway 10.0.1.1 --subnet 10.0.1.0/24 `
-    -o com.docker.network.windowsshim.interface="Ethernet0" external
+
+{{< note >}}
+The "pause" (infrastructure) image is on Microsoft Container Registry (MCR) and the DOCKERFILE is available at [https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/Dockerfile](https://github.com/Microsoft/SDN/blob/master/Kubernetes/windows/Dockerfile) 
+{{< /note >}}
+
+```powershell
+docker pull mcr.microsoft.com/k8s/core/pause:1.0.0
```

Comment:
This is a bit new to me... Does this image run on every Windows host OS version, and has it been validated to work seamlessly with the current scripts and the instructions? If not, this could break pods from entering running state when users try to schedule it.

Reply:
it should. this is a nanoserver pause image. i am not sure if it was validated, but i believe @craiglpeters  had someone run the instructions line by line to make sure it worked

Reply:
This is a multi-platform image built for 1809 and 1803 amd64